airflow/providers/qubole/operators/qubole_check.py

```diff
@@ -105,7 +105,7 @@ class QuboleCheckOperator(_QuboleCheckOperatorMixin, SQLCheckOperator, QuboleOpe
 
     """
 
-    template_fields: Iterable[str] = set(QuboleOperator.template_fields) | set(
+    template_fields: Sequence[str] = set(QuboleOperator.template_fields) | set(
```

Comment:
Expression is a set here too.

Reply:
Yeah. Good points @josh-fell  - I will replace all uses with tuples (including those that are using lists).